src/plugins/intel_gpu/src/kernel_selector/cl_kernels/dynamic_quantize_gpu_kv_cache.cl

```diff
@@ -64,6 +64,8 @@ KERNEL(dynamic_quantize_gpu_kv_cache)(
     // The innermost dimension is always processed in the loop inside the kernel
     const uint x = 0;
 
+    half grp_max = 0.001h;
+    half grp_min = 0.001h;
```

Comment:
What if max == min == 0.01 ? 
Then max == min still to be 0.01 
and divide by zero will still happen..

Reply:
As discussed, update to adjust max value to the minimum value for the simplest way.